installer/config.yml

```diff
@@ -317,6 +317,47 @@ stream:
         # > default is false
         mobile: false
 
+    hls:
+      - # Whether the output is enabled.
+        # > default is false
+        enabled: false
+        # > must be one of ('mpegts', 'mp3', 'adts', 'mp4')
+        # > this field is REQUIRED
```

Comment:
Let's reduce the amount of format for now, extra format can be added and tested in future PRs.

Reply:
AC3/ADTS is in the spec, further it work as it should with hls.js so i'll let it .

I'll remove mp4 because i haven't achieve anything with liquidsoap 1.4 even if in spec...:/

ok to remove mp3

I'll will move format property as stream property to fit liquidsoap stream model (it was a mistake to put it as hls property)